### PR TITLE
BENTO-2225

### PR DESCRIPTION
**Pull request by JoonLeeNIH**
## Description

Refactored the `StatsBar` component and place it in `src/bento-core` for it to be added to the `bento-core` library when the time comes to create the library.

Fixes #[BENTO-2225](https://tracker.nci.nih.gov/browse/BENTO-2225)

## Type of change

Should not change behavior.

## How Has This Been Tested?

The dashboard page works the same using this new `StatsBar` component as it did using the `StatsBar` component from `bento-components`.

### CODE REVIEW

File: src/components/Stats/StatsView.js [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
-    <StatsBar
-      data={data}
-      globalStatsData={globalStatsData}
-      statsStyling={statsStyling}
-    />
-  </>
-);
-
-export default StatsView;
+import React from 'react';
```
Comment (iksheth) [inline src/components/Stats/StatsView.js:1]: Are we using a Single Stats bar component on all the pages? Or Are we having variations of the Stats bar?

Comment (iksheth) [reply src/components/Stats/StatsView.js:1]: @JoonLeeNIH

Comment (JoonLeeNIH) [reply src/components/Stats/StatsView.js:1]: Just one `StatsBar` component. `bento-frontend` has the `StatsView` wrapper to pass props to `StatsBar`, and various files use `StatsView`.

**Review by iksheth — APPROVED** [review]
LGTM 👍🏻

**Review by rana22 — APPROVED** [review]
LGTM

**Pull request merged**